### ISSUE TITLE
[BUG] a11y issues

### ISSUE THREAD

**Issue by jonas-hoebenreich** [formbricks/formbricks#2627]
### Issue Summary

V2.0.1 introduced a couple of a11y issues:

1. The contrast between the star and the background is just 1.09:1 failing all WCAG standards (https://color.a11y.com/ContrastPair/?bgcolor=ffffff&fgcolor=f1f5f9)
![image](https://github.com/formbricks/formbricks/assets/64426524/3d9e65cb-74a6-4bdf-b630-5ad6b013f102)

2. The interactive area of the close icon is just 16x16px. This makes is hard to close the survey on mobile devices and does not comply with WCAG standards. see https://github.com/formbricks/formbricks/pull/1661

### Steps to Reproduce

1. check website survey for a11y

### Expected behavior

Website surveys comply with the Americans with Disabilities Act: https://www.ada.gov/resources/web-guidance/

### Environment

- [ ] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- v2.0.1
```
